util/cmds/render.go

```diff
@@ -0,0 +1,160 @@
+package cmds
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+var RespNil = "(nil)"
+var emptyList = "(empty list or set)"
+var invalidString = "(error) invalid type"
+
+// getRender retrieves the appropriate callback for the command
+func GetRender(commandName string) DiceDBCallback {
+	commandUpper := strings.ToUpper(strings.TrimSpace(commandName))
+	callback, exists := command2callback[commandUpper]
+	if !exists {
+		return renderListOrString
+	}
+
+	// If the type assertion fails, return a default fallback function
+	return callback
+}
+
+// Render functions
+
+func renderBulkString(value interface{}) interface{} {
+	if value == nil {
+		return RespNil
+	}
+	result, ok := value.(int64)
+	if ok {
+		return renderInt(result)
+	}
+	return (fmt.Sprintf("%v", value))
+}
+
+func renderInt(value interface{}) interface{} {
+	if value == nil {
+		return RespNil
+	}
+	if intValue, ok := value.(int64); ok {
+		return fmt.Sprintf("(integer) %d", intValue)
+	}
+	return invalidString
+}
+
+func renderList(value interface{}) interface{} {
+	items, ok := value.([]interface{})
+	if !ok {
+		return invalidString
+	}
+
+	var builder strings.Builder
+	for i, item := range items {
+		// Convert item to string
+		if item == nil {
+			builder.WriteString(fmt.Sprintf("%d) %v\n", i+1, RespNil))
+			continue
+		}
+		strItem := fmt.Sprintf("%v", item)
+
+		// Properly format with double quotes
+		quoted := fmt.Sprintf("\"%q\"", strItem)
```

Comment:
Can it just be single quotes? We're printing with 2 quotes? Is behaviour same as CLI?
```
dice > KEYS k*
1) ""k""
2) ""k1""
```